src/planners/rbt/RGBTConnect.cpp

```diff
@@ -97,20 +97,20 @@ std::tuple<base::State::Status, std::shared_ptr<std::vector<std::shared_ptr<base
 {
     float d_c = ss->computeDistance(q);
 	std::shared_ptr<base::State> q_new = q;
-	std::shared_ptr<std::vector<std::shared_ptr<base::State>>> q_new_list = std::make_shared<std::vector<std::shared_ptr<base::State>>>();
+	std::vector<std::shared_ptr<base::State>> q_new_list;
```

Comment:
Fix the size.

Reply:
The same reason, I don't know how many elements will be added to the vector.

Reply:
This has been already discussed in #13.